Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -1549,8 +1560,7 @@ bool RenderBox::hitTestClipPath(const HitTestLocation& hitTestLocation, const La
             break;
         if (!is<SVGClipPathElement>(*element))
             break;
-        CheckedRef clipper = downcast<LegacyRenderSVGResourceClipper>(*element->renderer());
```

Comment:
Oops, we lost the recent 'CheckedRef' introduction -- that needs to be fixed in your new hitsClipContent lambda!

Reply:
Done.